### ISSUE TITLE
Next release planning (4.2.6)

### ISSUE THREAD

**Issue by intgr** [typeddjango/django-stubs#1780]
As [announced in 4.2.5 release notes](https://github.com/typeddjango/django-stubs/releases/tag/4.2.5), we will drop the hard dependency on mypy. Users of django-stubs with mypy will need to add their own dependency on mypy, or install `django-stubs[compatible-mypy]` extra.

I'm hoping to release this in less than a week to fix a few bugs.

Blockers:

* #1778 (fixes a bug introduced in version 4.2.5)
* #1782
* #1784
* #1786

**Comment by intgr** [comment]
Nice to have, can we get someone to review this, it would unblock several other PRs:

* #1771

**Comment by intgr** [comment]
And it's out:

* Release notes: https://github.com/typeddjango/django-stubs/releases/tag/4.2.6
* PyPI: https://pypi.org/project/django-stubs/4.2.6/#history
